microsoft/PowerToys
# Added missed dlls

## Summary of the Pull Request

**What is this about:**
The PowerToys settings window fails to open on the current master because of missing dlls.

**What is include in the PR:** 

**How does someone test / validate:** 
- Build the installer
- Install 
- Open the settings window
- Verify that windows is opened

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
